meta-mender-core/recipes-mender/mender-server-certificate/mender-server-certificate.bb

```diff
@@ -0,0 +1,31 @@
+DESCRIPTION = "Mender self-signed server certificate"
+HOMEPAGE = "https://mender.io"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+DEPENDS_append = " ca-certificates"
+RDEPENDS_${PN} = "ca-certificates"
```

Comment:
I think both should be without `_append` and leading space. This is the base recipe, so it's the first definition.